about.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <link rel="stylesheet" href="./css/global.css">
+  <link rel="stylesheet" href="./css/navsection.css">
+  <link rel="stylesheet" href="./css/aboutsection.css">
+  
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+  <script src="https://kit.fontawesome.com/bdd8845187.js" crossorigin="anonymous"></script>
+  
+  <title>About MoP.</title>
+</head>
+<body>
+<main>
+  <header>
+    <nav class="navigation d-flex align-items-center justify-content-between">
+      <i class="fa-solid fa-bars open-menu"></i>
+      <a href="./index.html">MoP.™</a>
+      <ul class="navigation__list">
+        <li><a href="./index.html">Home</a></li>
+        <li><a href="index.html#program">Program</a></li>
+        <li><a href="index.html#speakers">Speakers</a></li>
+      </ul>
+    </nav>
+  </header>
+  <ul class="navigation__mobile">
+    <li><a href=""><img id="close-menu" class="close-menu" src="./images/close.png" alt="close"></a></li>
+     <li><a href="./index.html" class="navigation__mobile__links externalLink">Home</a></li>
+     <li><a href="index.html#program" class="navigation__mobile__links externalLink">Program</a></li>
+     <li><a href="index.html#speakers" class="navigation__mobile__links externalLink">Speakers</a></li>
+  </ul>
+  <section class="about">
+    <p class="about__para1 text-center">
+      "Greetings, students of Philosophy"
+    </p>
+    <h1>
+      Masters of Philosophy™ <br>
+      ANNUAL CONGRESS OF DISCOURSE <br>
+      2022 <br>
+    </h1>
+    <p class="about__para2">
+      The Congress brings together the most brilliant minds in modern Philosophy
+      to share knowledge and discuss about the latest advancements in the numerous 
+      fields of philosophy. We have philosphy professors from all over the globe 
+      who grace us with their attendance to give passionate lectures on the translations
+      of all our favorite ancient philosophers. 
+    </p>
+    <div class="p-3 text-center">
+      <h2 class="about__enq">
+        For any enquiries about the Congress of Discourse, 
+        feel free to contact us at:
+      </h2>
+      <a class="about__enq__mail" href="#">mastersofphilosophy@gmail.com</a>
+    </div>
+  </section>
+
```

Comment:
- [ ] [Optional] You should add a section here with a title and a logo check the original design for more information, kindly include this section in your design (Website)

![Screenshot_20220526_205057](https://user-images.githubusercontent.com/25227022/170567052-a6e20dcc-abec-43dd-aa2d-8132ba8352da.png)